docs/docs/mlflow-3/genai-agent.mdx

```diff
@@ -0,0 +1,201 @@
+        model_type="agent",
+        input_example={"messages": "What is MLflow?"},
+        # specify the prompts used, it will be linked to the run
+        prompts=[system_prompt],
+    )
+
+# Inspect the LoggedModel and its properties
+logged_model = mlflow.get_logged_model(model_info.model_id)
+print(logged_model.model_id, logged_model.params)
+# c981a134-0e42-4765-9e95-2d10f77023a8 {'max_tokens': '2000', 'temperature': '0.1'}
+```
+
+Check out the **Models** tab in the experiment to view the newly logged model named `chain` and its parameters.
+
+<div className="center-div" style={{ width: "100%" }}>
+    ![The MLflow UI showing the logged models in an experiment](/images/mlflow-3/genai/logged_models_tab.png)
+</div>
+
+On the **Logged Model** page, you can view detailed information, including the `model_id`, the source run of the model, and its parameters.
+
+<div className="center-div" style={{ width: "100%" }}>
+    ![The MLflow UI showing the logged model details page](/images/mlflow-3/genai/logged_model_page.png)
+</div>
+
+## Test the agent with tracing observability
+
+```python
+# Enable autologging so that interactive traces from the chain are automatically linked to its LoggedModel
+mlflow.langchain.autolog()
+
+loaded_chain = mlflow.langchain.load_model(model_info.model_uri)
+chain_inputs = [
+    "What is MLflow Tracking and how does it work?",
+    "What is Unity Catalog?",
+    "What are user-defined functions (UDFs)?",
+]
+
+for chain_input in chain_inputs:
+    loaded_chain.invoke({"messages": chain_input})
+
+traces = mlflow.search_traces(model_id=logged_model.model_id)
+print(traces)
+
+#                          request_id                                              trace   timestamp_ms  ...                                              spans                                               tags assessments
+# 0  c4922d6515004439ba6745506969c401  Trace(request_id=c4922d6515004439by6745506969c...  1742786913864  ...  [{'name': 'RunnableSequence', 'context': {'spa...  {'mlflow.artifactLocation': 'file:///Users/ser...          []
+# 1  a6e4de782b5540f1af2f2661ba85abd2  Trace(request_id=a6e4de782b5540f1af2f2661by85a...  1742786909657  ...  [{'name': 'RunnableSequence', 'context': {'spa...  {'mlflow.artifactLocation': 'file:///Users/ser...          []
+# 2  bc72367e96ae4e978e28594d5ce438e8  Trace(request_id=bc72367e96ae4e978e28594d5ce43...  1742786905377  ...  [{'name': 'RunnableSequence', 'context': {'spa...  {'mlflow.artifactLocation': 'file:///Users/ser...          []
+```
+
+Navigating to the **Traces** tab of the model's page, you can view the traces that were just generated.
+
+<div className="center-div" style={{ width: "100%" }}>
+    ![The MLflow UI showing the logged model autolog traces lineage](/images/mlflow-3/genai/logged_model_autolog_traces.png)
+</div>
+
+## Evaluate on the agent
+
+Then we use MLflow evaluate to assess the performance of the logged model on the evaluation dataset. 
```

Comment:
```suggestion
Finally, we use `mlflow.evaluate()` to assess the performance of the logged model on an evaluation dataset. 
```

Reply:
Can we make this an  <APILink>? (I added backticks instead just to communicate the desired phrasing)